x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/lib/check_missing_group.ts

```diff
@@ -69,9 +69,9 @@ export const checkMissingGroups = async (
     ];
   });
 
-  logger.trace(`Request: ${JSON.stringify({ searches })}`);
+  logger.trace(() => `Request: ${JSON.stringify({ searches })}`);
```

Comment:
Created a ticket for improving logging msearch requests and responses: https://github.com/elastic/kibana/issues/192445